label_studio_converter/converter.py

```diff
@@ -900,6 +900,10 @@ def _get_labels(self):
         category_name_to_id = dict()
 
         for name, info in self._schema.items():
+            # ignore labels that are not bounding boxes or segmentations
```

Comment:
Sorry for a long review. 
1. We have to add `rectangle` and `polygon` too - https://labelstud.io/tags/rectangle.html
2. `_get_labels` is used in COCO also, COCO supports another set of labels.

Reply:
This means that there should be a different _get_labels procedure for YOLO and for COCO, right? Because they support different kinds of labels.

Reply:
I would pass all supported types this way  
```
_get_labels(types=['rectanglelabels', ...])
```
and used different calls in YOLO and COCO.

Reply:
One more question - if _get_labels returns nothing, will the image be included to export?